docs/source/playbook.rst

```diff
@@ -0,0 +1,191 @@
+Writing actions
+===============
+
+All Ansible is contained in `obal/data`. There we have `playbooks` and `roles`.
+
+A `playbook` with `metadata` is considered an action and exposed to the user as such.
+
+Writing playbooks
+-----------------
+
+We have a slightly non-standard playbooks layout. Every playbook is contained in its own directory and named `playbook.yaml`. It can also contain a `metadata.yaml`. While playbooks are pure ansible, the metadata is the data Obal needs to extract to build a CLI.
```

Comment:
Did we settle on `playbook.yaml`? I generally find similarly named files in different directories to be aggravating when developing. When trying to find the file one is looking for manually or using an IDE to search for it I find this slows things down. I tend to find it's also not very descriptive. This *can* be redundant given the directory name is intended to specify the name, however, I prefer that redundancy.

Reply:
I like:

```
+- obal
  +- data
    +- playbooks
      +- changelog
        +- changelog.yaml
        +- metadata.yaml
      +- release
      [...]
```